demo/demo6.html

```diff
@@ -0,0 +1,128 @@
+  <link rel="import" href="../app-drawer-layout/app-drawer-layout.html">
+  <link rel="import" href="../app-drawer/app-drawer.html">
+  <link rel="import" href="../app-scroll-effects/app-scroll-effects.html">
+  <link rel="import" href="../app-header/app-header.html">
+  <link rel="import" href="../app-header-layout/app-header-layout.html">
+  <link rel="import" href="../app-toolbar/app-toolbar.html">
+  <link rel="import" href="../demo/sample-content.html">
+
+  <style is="custom-style">
+
+    body {
+      margin: 0;
+      font-family: 'Roboto', 'Noto', sans-serif;
+      background-color: #eee;
+    }
+
+    app-header {
+      background-color: #4285f4;
+      color: #fff;
+    }
+
+    app-header paper-icon-button {
+      --paper-icon-button-ink-color: #fff;
+    }
+
+    app-drawer-layout {
+      --app-drawer-layout-content-transition: margin 0.2s;
+    }
+
+    app-drawer {
+      --app-drawer-content-container: {
+        background-color: #eee;
+      }
+    }
+
+    .drawer-content {
+      margin-top: 80px;
+      height: calc(100% - 80px);
+      overflow: auto;
+    }
+
+  </style>
+
+</head>
+<body unresolved>
+
+  <app-header-layout>
+
+    <app-drawer-layout id="drawerLayout">
+
+      <app-drawer>
+        <div class="drawer-content">
+          <paper-icon-item>
+            <iron-icon icon="inbox" item-icon></iron-icon> <span>Inbox</span>
+          </paper-icon-item>
+          <paper-icon-item>
+            <iron-icon icon="query-builder" item-icon></iron-icon> <span>Snoozed</span>
+          </paper-icon-item>
+          <paper-icon-item>
+            <iron-icon icon="done" item-icon></iron-icon> <span>Done</span>
+          </paper-icon-item>
+          <paper-icon-item>
+            <iron-icon icon="drafts" item-icon></iron-icon> <span>Drafts</span>
+          </paper-icon-item>
+          <paper-icon-item>
+            <iron-icon icon="send" item-icon></iron-icon> <span>Sent</span>
+          </paper-icon-item>
+          <paper-icon-item>
+            <iron-icon icon="delete" item-icon></iron-icon> <span>Trash</span>
+          </paper-icon-item>
+          <paper-icon-item>
+            <iron-icon icon="report" item-icon></iron-icon> <span>Spam</span>
+          </paper-icon-item>
+        </div>
+      </app-drawer>
+      
+      <sample-content size="100"></sample-content>
+
+    </app-drawer-layout>
+
+    <app-header fixed effects="waterfall">
+      <app-toolbar>
+        <paper-icon-button id="toggle" icon="menu"></paper-icon-button>
+        <div title>Inbox</div>
+      </app-toolbar>
+    </app-header>
+
+  </app-header-layout>
+
+  <script>
+    var drawerLayout = document.getElementById('drawerLayout');
+    var drawer = drawerLayout.drawer;
+
+    document.getElementById('toggle').addEventListener('tap', function() {
+      if (drawerLayout.forceNarrow || !drawerLayout.narrow) {
+        drawerLayout.forceNarrow = !drawerLayout.forceNarrow;
+      } else {
+        drawer.toggle();
```

Comment:
`drawer` would be undefined on WebComponents polyfill browser (e.g. Safari).  You can either wrap the whole thing inside `HTMLImports.whenReady` or simply just change the above line to `drawerLayout.drawer.toggle()`